src/codegen/llvm.zig

```diff
@@ -8588,7 +8588,10 @@ pub const FuncGen = struct {
         const src_slice = try self.resolveInst(bin_op.rhs);
         const src_ptr_ty = self.air.typeOf(bin_op.rhs);
         const src_ptr = self.sliceOrArrayPtr(src_slice, src_ptr_ty);
-        const len = self.sliceOrArrayLenInBytes(dest_slice, dest_ptr_ty);
```

Comment:
AIR docs for memcpy say:
> Dest pointer is either a slice or a pointer to array.

Reply:
The language reference says that a many pointer is allowed https://ziglang.org/documentation/master/#memcpy, should that be changed?

Reply:
no - a many pointer is allowed in the language but semantic analysis will ensure the length property will be communicated always by the dest argument.